packages/svelte/src/internal/client/runtime.js

```diff
@@ -59,6 +59,11 @@ let current_queued_pre_and_render_effects = [];
 /** @type {import('./types.js').EffectSignal[]} */
 let current_queued_effects = [];
 
+// Tracks the current property access to a derived signal object, e.g.
+// `a.b.c` for each property access, we'd get on the derived proxy object `a`.
+/** @type {{ s: import('./types.js').ComputationSignal, p: Array<string | symbol> } | null} */
```

Comment:
What is meant by "we'd get on"?
```suggestion
/**
 * Tracks the current property access to a derived signal object, e.g.
 * for each property access in `a.b.c`, we'd get on the derived proxy object `a`.
 * @type {{ s: import('./types.js').ComputationSignal, p: Array<string | symbol> } | null}
 */
```

Reply:
I suggested change but the change wasn't meant to be final. I still don't fully understand the sentence, especially what "we'd get on" is refering to.

Reply:
I know, I pushed a commit to fix and marked this as resolved. :D